book/website/communication/citable/citable-orcid.md

```diff
@@ -0,0 +1,33 @@
+(rr-credit-orcid)=
+# ORCID to Recognise Researchers
```

Comment:
Not sure the title is clear - I think it's less about recognition and more about making it easier for you to collate/link all your citable material together?